portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,3 @@
 // See the License for the specific language governing permissions and
```

Comment:
I'd say you can delete this whole file and remove the reference to it from index.html

Reply:
+1